mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/entity/EntityHelperSetMethodGenerator.java

```diff
@@ -63,6 +65,7 @@ public EntityHelperSetMethodGenerator(
             .addTypeVariable(settableT)
             .addParameter(ParameterSpec.builder(entityDefinition.getClassName(), "entity").build())
             .addParameter(ParameterSpec.builder(settableT, "target").build())
+            .addParameter(ParameterSpec.builder(nullSavingStrategyT, "nullSavingStrategy").build())
```

Comment:
You don't need to explicitly build a type name, the builder accepts a simple class:
```suggestion
            .addParameter(ParameterSpec.builder(NullSavingStrategy.class, "nullSavingStrategy").build())
```
(I only did it for `SettableT` because it's more complex)